### ISSUE TITLE
mAP increase on COCO after training 300 epochs

### ISSUE THREAD

**Issue by abhiksark** [ultralytics/yolov5#10701]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I am training [YOLOv5l](https://github.com/ultralytics/yolov5/releases/download/v6.2/yolov5l.pt) on COCO taking the weights which were shared. 
I noticed that the mAP_0.5  increases to 65% on COCO after training for 300 epochs. Can someone explain why this is happening?
If that is the case, what should be ideal number of Epoch to get best result in COCO

I don't suspect Overfitting as both train and val loss are decreasing. [image](l)

### Additional

<a href="https://gcdnb.pbrd.co/images/r0b14a2L550O.png?o=1"><img src="https://gcdnb.pbrd.co/images/r0b14a2L550O.png?o=1" alt="Loss" border="0"></a>

**Comment by skro123** [comment]
@abhiksark  你好，我也发现了类似的问题。我使用固定学习率0.001和AdamW优化器，训练超过300epoch后，map仍然在增长，但奇怪的是验证集的三个loss也都在增长。总结就是验证集的map和loss都在增长。使用pycocotools计算的结果与使用val.py计算的结果相差很大。版本6.2出现了这个问题，但版本6.0是正常的。
![8e858b9b768d2c58bdb3f133f63d156](https://user-images.githubusercontent.com/35415749/231142084-1406e870-c52b-4c6c-9f4f-04b4ecf21a82.png)
![ffcaf536df7e0c5723433c63cab97ed](https://user-images.githubusercontent.com/35415749/231142116-23d8cd1c-de8c-459e-bf4d-bf57c36fdfe0.png)









Model summary: 290 layers, 20865057 parameters, 0 gradients, 47.9 GFLOPs
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100%|██████████| 90/90 [01:16<00:00,  1.18it/s]                    
                 all       1800      13022      0.882      0.759      0.846      0.527
         holothurian       1800       1603      0.862      0.708      0.795      0.473
             echinus       1800       6327      0.907      0.826      0.898      0.537
             scallop       1800       2142      0.867      0.722      0.822      0.535
            starfish       1800       2950      0.892      0.778       0.87      0.563
Speed: 0.2ms pre-process, 7.2ms inference, 4.7ms NMS per image at shape (20, 3, 640, 640)

 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.478
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.807
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.516
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.205
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.409
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.536
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.177
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.504
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.556
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.313
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.501
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.603
